esp/scm/ws_logaccess.ecm

```diff
@@ -229,10 +229,59 @@ ESPResponse GetLogsResponse
     [min_ver("1.02")] unsigned int TotalLogLinesAvailable;
 };
 
-ESPservice [auth_feature("WsLogAccess:READ"), version("1.06"), default_client_version("1.06"), exceptions_inline("xslt/exceptions.xslt")] ws_logaccess
+ESPRequest GetHealthReportRequest
+{
+    bool IncludeConfiguration(false);
+    bool IncludeDebugReport(false);
+    bool IncludeSampleQuery(false);
+};
+
+ESPenum LogAccessStatusCode : int
```

Comment:
@jeclrsg this service can either provide a green/yellow/red type status, or more specific status codes as in the enum below. Let me know what you'd prefer.